readme.md

```diff
@@ -138,12 +225,56 @@ Content-Length: 112
 ]
 ```
 
-####If the course doesn’t exist:
+#####If data is invalid:
 
 ```
 $ nc localhost 8008 << HERE
 >
 > GET /course/1117 HTTP/1.0
+> POST /course HTTP/1.0
```

Comment:
looks like GET and POST were incorrectly merged here as well.

it might be easier just to run these requests again and paste in the output.